### ISSUE TITLE
IzPack Installer

### ISSUE THREAD

**Issue by agazzarini** [agazzarini/SolRDF#76]
The installation of a sample SolRDF is too much technical, regardless the two ways described in the "Get me up and running" page. 
   
Building things from source [1] is useful in case you've a bit of familiarity with Java, Maven: it will download all required software and dependencies but at the same time you could meet some annoying Maven related things. This is probably what happened to the user who opened the issue #75.   
      
Installing the download bundle [2] \(Solr home + dependencies\) provided in the GitHub still requires some manual step and that could be a boring thing for a non technical user who would like simply to have a quick and first taste of SolRDF.  

IzPack could be an ideal choice for supplying an easy installer for a sample SolRDF instance. The installer is a single jar that will automatize the whole installation process.           

----------------
[1] https://github.com/agazzarini/SolRDF/wiki/User%20Guide#if-you-dont-have-solr-installed    
[2] https://github.com/agazzarini/SolRDF/wiki/User%20Guide#if-you-already-have-solr-installed

**Comment by ahagenbruch** [comment]
Maybe using Docker could be another route for us to investigate. Perhaps we should look into how the Solr-Dockerfile https://github.com/makuk66/docker-solr has to be modified to make things work for SolRDF...

**Comment by agazzarini** [comment]
Yes yes absolutely, this morning Martin send a post about this thing and 
I completely agree. I don't know IzPack and that "non functional" thing 
would require some time. Instead using a VM image or something like that 
could be the best option, as it's just matter to install the software 
and put the VM somewhere (I  created a dropbox account so we could use 
links coming from public folder)
